index.d.ts

```diff
@@ -0,0 +1,32 @@
+export declare type MachineState = 'running';
+export declare type Driver = 'amazonec' | 'azure' | 'digitalocean' | 'exoscale' | 'g5k' | 'google' | 'hetzner' | 'hyperv' | 'linode' | 'openstack' | 'rackspace' | 'scaleway' | 'softlayer' | 'virtualbox' | 'vmwarefusion' | 'vmwarevcloudair' | 'vmwarevsphere' | 'vmwareworkstation' | 'vmwareworkstation' | '';
```

Comment:
I'm not sure it's a good idea to define these values here, because they are not tied to our semver-version.

Reply:
Do you refer to `Driver` or `MachineState` or both?
What would be your suggestions and which problems do you expect?

Reply:
> Do you refer to `Driver` or `MachineState` or both?

`Driver`.

> What would be your suggestion?

Only enforce it to be a string, don't check values.

> Which problems do you expect?

Having to make a release when a new driver comes out.

Reply:
I'm trying to find another solution.